src/main/java/util/GitHubURL.java

```diff
@@ -46,11 +50,11 @@ public static String getPathForContributors(String repoId) {
     }
 
     public static boolean isUrlIssue(String url) {
-        return url.matches("https://github.com/([^/]+)/([^/]+)/(issues|pull)/([0-9]+)([/commits,/files]*)");
+        return url.matches("https://github\\.com/([^/]+)/([^/]+)/(issues|pull)/([0-9]+)([/commits,/files]*)");
     }
```

Comment:
Actually, the confusion comes from the word 'issue' and it's not a result of your code. Does 'issues' include PRs? If it does, why do we have a method called `isUrlIssueOrPrDiscussionPage`? If they are mutually exclusive, this method should be `isUrlPrOrIssue`?